mock/py/mockbuild/shadow_utils.py

```diff
@@ -13,16 +15,35 @@ class ShadowUtils:
     """
     def __init__(self, root):
         self.root = root
+        self._selinux_workaround_applied = False
+
+    def _selinux_workaround(self):
+        """
+        Work-around for:
+        https://github.com/shadow-maint/shadow/issues/940
+        https://github.com/SELinuxProject/selinux/issues/419
+        """
+        if self._selinux_workaround_applied:
+            return
+
+        path = self.root.make_chroot_path("/sys/fs/selinux")
+        mkdirIfAbsent(path)
+        file = os.path.join(path, "enforce")
+        with open(file, "w", encoding="utf-8") as fd:
+            fd.write("0")
+        self._selinux_workaround_applied = True
 
     def _execute_command(self, command, can_fail=False):
+        self._selinux_workaround()
```

Comment:
When the fix from https://github.com/shadow-maint/shadow/pull/1258 gets released, we no longer need the `_selinux_workaround()`.